extension/threadpool/CMakeLists.txt

```diff
@@ -21,7 +21,8 @@ if(NOT CMAKE_CXX_STANDARD)
 endif()
 
 add_library(
-  extension_threadpool threadpool.cpp threadpool_guard.cpp cpuinfo_utils.cpp
+  extension_threadpool threadpool.cpp threadpool_guard.cpp thread_parallel.cpp
```

Comment:
nit: ok with this but thread_parallel provides interface to leverage underlying threadpool.

Reply:
I don't understand your point here. I can say that I originally left extension_parallel around, but it struck me as silly to make code depend on thread_parallel separately rather than getting it for free as a way to use the thread pool.

Reply:
> make code depend on thread_parallel separately rather than getting it for free as a way to use the thread pool

That also makes sense. I was making a bit of a pedantic point but no value in arguing over it.